packages/ti_domaintools/manifest.yml

```
@@ -0,0 +1,32 @@
+format_version: 3.3.0
+name: ti_domaintools
+title: "DomainTools Real Time Unified Feeds"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "The DomainTools NOD Feed provides real-time access to newly registered and observed domains, enabling proactive threat detection and defense."
+type: integration
+categories:
+  - security
+  - threat_intel
+conditions:
+  kibana:
+    version: "^8.15.3"
+  elastic:
+    subscription: "basic"
+icons:
+  - src: /img/logo.png
+    title: DomainTools logo
+    size: 250x250
+    type: image/svg+xml
+policy_templates:
+  - name: domaintools
+    title: DomainTools Newly Observed Domains
+    description: "DomainTools Newly Observed Domains provides real-time access to newly registered and observed domains, enabling proactive threat detection and defense"
+    inputs:
+      - type: cel
+        title: "DomainTools Newly Observed Domains"
+        description: "DomainTools Newly Observed Domains provides real-time access to newly registered and observed domains, enabling proactive threat detection and defense"
+owner:
+  github: elastic/integrations
```

```suggestion
  github: elastic/security-service-integrations
```

https://github.com/elastic/integrations/blob/main/packages/ti_recordedfuture/manifest.yml#L41

---

done!